config.libsonnet

```diff
@@ -1,13 +1,33 @@
+// Core configuration (mostly from upstream)
+(import 'grafana/grafana.libsonnet') +
+(import 'node-mixin/mixin.libsonnet') +
+(import 'prometheus-operator/prometheus-operator.libsonnet') +
+//(import 'prometheus-adapter/prometheus-adapter.libsonnet') +
+(import 'kube-prometheus/alerts/alerts.libsonnet') +
+(import 'kube-prometheus/rules/rules.libsonnet') +
+(import 'kubernetes-mixin/mixin.libsonnet') +
+(import 'prometheus/mixin.libsonnet') +
+(import 'lib/prometheus.libsonnet') +
+(import 'lib/grafana.libsonnet') +
+(import 'lib/mintel/rules.libsonnet') +
```

Comment:
shoud we keep the previous "mixin.libsonnet" and import that so to keep the logic out of this file ?